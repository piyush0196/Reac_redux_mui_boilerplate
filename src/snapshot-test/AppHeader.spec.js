import { theme } from 'theme/theme'
import { ThemeProvider } from '@mui/material'
import TestRenderer from 'react-test-renderer'
import AppHeader from 'components/ui-kit/AppHeader'

describe('Component - AppHeader', () => {
  const defaultProps = {
    openDrawer: true,
    handleDrawer: () => {},
    left: <h6>Left Side Content</h6>,
    right: <h6>Right Side Content</h6>
  }

  it('renders AppHeader component correctly', () => {
    const wrapper = TestRenderer.create(
      <ThemeProvider theme={theme}>
        <AppHeader {...defaultProps} />
      </ThemeProvider>
    ).toJSON()
    expect(wrapper).toMatchSnapshot()
  })
})
