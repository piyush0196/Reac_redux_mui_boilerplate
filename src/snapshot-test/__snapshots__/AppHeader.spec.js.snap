// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component - AppHeader renders AppHeader component correctly 1`] = `
<header
  className="MuiPaper-root MuiPaper-elevation MuiPaper-elevation4 MuiAppBar-root MuiAppBar-colorDefault MuiAppBar-positionFixed mui-fixed css-doe4zy-MuiPaper-root-MuiAppBar-root"
>
  <div
    className="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular css-yxz43g-MuiToolbar-root"
  >
    <div>
      <h6>
        Left Side Content
      </h6>
    </div>
    <div>
      <h6>
        Right Side Content
      </h6>
    </div>
  </div>
</header>
`;
